assets/environment-interpreter/activate/activate.d/start.bash

```diff
@@ -39,6 +39,9 @@ start() {
   if [ "$_FLOX_ENV_ACTIVATION_MODE" = "dev" ] || [ "$_FLOX_ENV_ACTIVATION_MODE" = "build" ]; then
     # shellcheck disable=SC1090 # from rendered environment
     source_profile_d "$_profile_d"
+  else
+    # shellcheck disable=SC1091 # from rendered environment
+    source "$_profile_d/0100_common-run-mode-paths.sh"
```

Comment:
(reviewer commentary) I thought we already sourced a run-mode specific `profile.d` script which might also need updating but it turned out I was remembering a previous iteration of https://github.com/flox/flox/pull/2342 and before we then moved the PATH/MANPATH into `flox-activations`.